README.rst

```diff
@@ -29,3 +29,6 @@ follows::
   $ pip install pytest
   $ py.test tests
 
+Nota bene: for testing locally I had to run the following commands::
```

Comment:
remove this and change this
`$ python2 -m virtualenv env`
to
`$ virtualenv -p YOUR_PYTHON env`